src/default_client.rs

```diff
@@ -327,6 +422,24 @@ impl<A: ClientAdapter + Sync + Send> Client for DefaultClient<A> {
             .map(DefaultClient::<A>::map_session)
     }
 
+    /// Authenticate a user with Apple Game Center
+    ///
+    /// See [Game center](https://heroiclabs.com/docs/authentication/#game-center) on how to set up authentication using the Apple Game Center.
+    /// todo! The documentation doesn't help here
```

Comment:
Is there any documentation on how to use game center in Nakama?

Reply:
Yep. Here: https://heroiclabs.com/docs/authentication/#game-center

Reply:
It doesn't explain how to retrieve the `bundleid `values etc. Should we ignore that for now?

Reply:
Yeah you don't need to explain where any external values come from for now.